src/Traits/LogsActivity.php

```diff
@@ -16,10 +16,17 @@ protected static function bootLogsActivity()
     {
         static::eventsToBeRecorded()->each(function ($eventName) {
             return static::$eventName(function (Model $model) use ($eventName) {
-                if ($eventName != 'deleted' && (! count(array_except($model->getDirty(), $model->attributesToBeIgnored())))) {
+
+                if ($eventName != 'restored' && $eventName != 'deleted' && (! count(array_except($model->getDirty(), $model->attributesToBeIgnored())))) {
```

Comment:
This check is unreadable. Could you split this out into multiple if's with an early return. A dedicated well named function for this check would also be ok.